.github/workflows/browser-test.yml

```diff
@@ -41,10 +53,10 @@ jobs:
         run: npm ci --ignore-scripts
       - name: Install Playwright Browsers
         working-directory: browser-test
-        run: npx playwright install --with-deps
+        run: npm exec playwright install --with-deps
```

Comment:
Doesn't `npx` do the same thing if `playwright` is local?

Reply:
atm should be

```suggestion
        run: npm exec -- playwright install --with-deps
```